### PR TITLE
Some deabbreviations

### PR DESCRIPTION
**Pull request by beldmit**
Some abbreviations in x509 manual page are hard to either understand or to google.
##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

**Comment by levitte** [comment]
Hmmm, do we need a glossary?

**Comment by beldmit** [comment]
I do not think we need a glossary, but these abbreviations were unclear.

File: doc/man1/x509.pod [hunk 1]
```diff
@@ -578,7 +578,8 @@ Dump any field whose OID is not recognised by OpenSSL.
 B<sep_multiline>
 
 These options determine the field separators. The first character is
-between RDNs and the second between multiple AVAs (multiple AVAs are
+between Relative Distinguised Names (RDNs) and the second is between
```
Comment (mspncp) [inline doc/man1/x509.pod]: Distinguished

Comment (beldmit) [reply doc/man1/x509.pod]: Fixed.

**Comment by paulidale** [comment]
Merged to master.  I took the liberty of removing an end of line space character in the process.

File: doc/man1/x509.pod [hunk 2]
```diff
@@ -578,7 +578,8 @@ Dump any field whose OID is not recognised by OpenSSL.
 B<sep_multiline>
 
 These options determine the field separators. The first character is
-between RDNs and the second between multiple AVAs (multiple AVAs are
+between Relative Distinguished Names (RDNs) and the second is between 
+multiple Attribute Value Assertions (AVAs, multiple AVAs are
```
Comment (sam-github) [inline doc/man1/x509.pod]: I suggest `(AVAs). Multiple ` Basically, don't hide the second sentence in parentheses.

**Pull request closed without merging**